osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -40,9 +43,14 @@ public class TaikoPlayfield : ScrollingPlayfield
 
         private Container hitTargetOffsetContent;
 
+        private readonly LayoutValue<float> playfieldScaleLayout = new LayoutValue<float>(Invalidation.DrawSize);
+        private float playfieldScale => playfieldScaleLayout.IsValid ? playfieldScaleLayout.Value : playfieldScaleLayout.Value = DrawHeight / DEFAULT_HEIGHT;
```

Comment:
is this really necessary for just saving on one divide? seems like way too much unnecessary complexity unless i'm missing something.

Reply:
Maybe was a little overeager to use that.